src/app/modules/registration/components/snow/exposed-height/exposed-height.component.ts

```diff
@@ -25,40 +28,10 @@ export class ExposedHeightComponent implements OnInit {
   exposedHeightMiddle: boolean;
   exposedHeightBottom: boolean;
 
-  heightArray: SelectOption[] = [
-    { id: 2500, text: '2500' },
-    { id: 2400, text: '2400' },
-    { id: 2300, text: '2300' },
-    { id: 2200, text: '2200' },
-    { id: 2100, text: '2100' },
-    { id: 2000, text: '2000' },
-    { id: 1900, text: '1900' },
-    { id: 1800, text: '1800' },
-    { id: 1700, text: '1700' },
-    { id: 1600, text: '1600' },
-    { id: 1500, text: '1500' },
-    { id: 1400, text: '1400' },
-    { id: 1300, text: '1300' },
-    { id: 1200, text: '1200' },
-    { id: 1100, text: '1100' },
-    { id: 1000, text: '1000' },
-    { id: 900, text: '900' },
-    { id: 800, text: '800' },
-    { id: 700, text: '700' },
-    { id: 600, text: '600' },
-    { id: 500, text: '500' },
-    { id: 400, text: '400' },
-    { id: 300, text: '300' },
-    { id: 200, text: '200' },
-    { id: 100, text: '100' },
-    { id: 0, text: '0' }
-  ];
-
-  get lowerHeightArray() {
-    return this.heightArray.filter(
-      (x) => this.exposedHight1 === undefined || x.id < this.exposedHight1
-    );
-  }
+  exposedHeight1Min: number = HEIGHT_MIN + 100;
```

Comment:
Ser du har en slags validering for å unngå at nedre høyde er over øvre høyde, men jeg greide faktisk å lure den:
![image](https://user-images.githubusercontent.com/71138449/143194366-e9a8689d-7b2d-4e07-baaa-dea22c1c2948.png)
Det er "bare" å prøve å skrive inn verdien på nytt etter at du har skrevet inn en "ulovlig" verdi. Sånn gjorde jeg det:

1. Skrev inn nedre høyde på 500
2. Skrev inn øvre høyde på 400. Denne ble så korrigert automatisk til 600
3. Korrigerte øvre høyde til 400. Da fikk jeg lov å lagre.
Øvre høyde blir igjen autokorrigert til 600 når du lukker skjemaet, men det ser du ikke.
Jeg synes det er litt uheldig at vi korrigerer verdier uten at observatøren ser det.